apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PlanUpdateSidePanel.tsx

```diff
@@ -321,73 +338,159 @@ const PlanUpdateSidePanel = () => {
+                  </Table.th>,
+                  <Table.th key="header-unit-price">Unit Price</Table.th>,
+                  <Table.th key="header-price" className="text-right">
+                    Cost
+                  </Table.th>,
+                ]}
+                body={
+                  <>
+                    {subscriptionPreview.breakdown.map((item) => (
                       <>
-                        {subscriptionPreview.breakdown.map((item) => (
-                          <Table.tr key={item.description}>
-                            <Table.td>{item.description ?? 'Unknown'}</Table.td>
-                            <Table.td>{item.quantity}</Table.td>
-                            <Table.td>
-                              {item.unit_price === 0 ? 'FREE' : `$${item.unit_price}`}
-                            </Table.td>
-                            <Table.td className="text-right">${item.total_price}</Table.td>
-                          </Table.tr>
-                        ))}
-
-                        <Table.tr>
-                          <Table.td>Total</Table.td>
-                          <Table.td />
-                          <Table.td />
-                          <Table.td className=" text-right">
-                            $
-                            {Math.round(
-                              subscriptionPreview.breakdown.reduce(
-                                (prev, cur) => prev + cur.total_price,
-                                0
-                              )
-                            ) ?? 0}
+                        <Table.tr key={item.description}>
+                          <Table.td>
+                            {item.breakdown && item.breakdown.length > 0 && (
+                              <Button
+                                type="text"
+                                className="!pl-0 !pr-1"
+                                icon={
+                                  <IconChevronRight
+                                    className={clsx(
+                                      'transition',
+                                      usageFeesExpanded.includes(item.description) && 'rotate-90'
+                                    )}
+                                  />
+                                }
+                                onClick={() =>
+                                  usageFeesExpanded.includes(item.description)
+                                    ? collapseUsageFee(item.description)
+                                    : expandUsageFee(item.description)
+                                }
+                              />
+                            )}
+                            {item.description ?? 'Unknown'}{' '}
+                            {item.description.endsWith('Compute') && ` (Hours)`}
+                          </Table.td>
+                          <Table.td className="text-right pr-4 tabular-nums">
+                            {item.quantity}
```

Comment:
should we do this? to format for e.g 1234 -> 1,234

```suggestion
                            {item.quantity.toLocaleString}
```